packages/rich-text/src/helpers/__tests__/removeInternalMarks.test.tsx

```diff
@@ -11,58 +11,58 @@ describe('internal mark', () => {
     const data = [
       {
         title: 'Paragraph mark is removed',
-        input: toContentfulDocument(
-          (
+        input: toContentfulDocument({
+          document: (
```

Comment:
Could you please explain why you now need to nest everything in a document?

Reply:
It’s the format that `toContentfulDocument` expects according to its types which were unavailable in the old `contentful-slatejs-adapter` version.

Reply:
Here’s the error i get when using the old format with the new `contentful-slatejs-adapter` version.
<img width="931" alt="image" src="https://github.com/contentful/field-editors/assets/1296628/38c9a7e4-5302-4b09-8d9d-5846c52a006e">

Reply:
Okay cool thanks for letting me know